pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -565,6 +565,7 @@ spec:
   serviceAccountName: test-sa
   taskRef:
     name: unit-test-task
+    kind: Task
```

Comment:
What's the reason to add `kind: Task` to all of these taskRefs?

Reply:
setdefaults will add kind for task as well, if not adding them the current tests will fail